packages/proxy/lib/http/response-middleware.ts

```diff
@@ -240,7 +240,11 @@ const MaybeDelayForMultiDomain: ResponseMiddleware = function () {
       this.next()
     })
 
-    this.serverBus.emit('cross:domain:delaying:html')
+    const url = new URL(`${this.req.protocol}://${this.req.get('host')}${this.req.originalUrl}`)
```

Comment:
Are you able to use `this.req.proxiedUrl`?

Reply:
TIL that it's a thing, I'll check it out!

Reply:
Looks like it'll work great, thanks for mentioning it.